src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperOptionsCollectionExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.TagHelpers;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for <see cref="ITagHelperOptionsCollection"/>.
+    /// </summary>
+    public static class TagHelperOptionsCollectionExtensions
+    {
+        /// <summary>
+        /// Configures options for the <see cref="FormTagHelper"/> from an <see cref="IConfiguration"/>.
+        /// </summary>
+        /// <param name="collection">The <see cref="ITagHelperOptionsCollection"/> instance this method extends.</param>
+        /// <param name="configuration">An <see cref="IConfiguration"/> to get the options from.</param>
+        /// <returns>The <see cref="ITagHelperOptionsCollection"/>.</returns>
+        public static ITagHelperOptionsCollection ConfigureForm(
```

Comment:
suggest these extension methods might be overkill for a single `bool?` property.  in MVC for example we've provided extension methods for the various descriptor collections but not for `AntiForgeryOptions` or its members.  or, do we plan to extend the rest of the system to support a fluent configuration API?